Newsletter2Go/Export/Model/Observer/RegisterIntegration.php

```diff
@@ -101,5 +105,4 @@ protected function revokePreviousTokens($activeToken)
 
         return $connection->update($resource->getMainTable(), ['revoked' => 1], $where);
```

Comment:
Note that the `$where` composed above is the source of SQL injection. Not a big deal since the user cannot control it, however an extra single quote (or some special character) would still cause issues in the query.